JPS_VIRTUALSENSOR/AermodAgent/docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: "3.8"
 
 services:
   aermod-agent:
-    image: ghcr.io/cambridge-cares/aermod-agent:1.6
+    image: ghcr.io/cambridge-cares/aermod-agent:1.6-dev-ship-demo-SNAPSHOT
```

Comment:
I believe the convention is to increase the version number and append the branch name, just remember to increase the version number before merging back

Reply:
Now changed to 1.7-dev-ship-demo-SNAPSHOT.